Source/Krypton Components/Krypton.Ribbon/Group Contents/KryptonRibbonGroupTrackBar.cs

```diff
@@ -412,20 +413,20 @@ public int Maximum
         [DefaultValue(0)]
         public int Minimum
         {
-            get => TrackBar.Minimum;
-            set => TrackBar.Minimum = value;
+            get => TrackBar!.Minimum;
+            set => TrackBar!.Minimum = value;
         }
 
         /// <summary>
         /// Gets or sets a numeric value that represents the current position of the scroll box on the track bar.
         /// </summary>
         [Category(@"Behavior")]
-        [Description(@"Current position of the indicator within the trackbar.")]
+        [Description(@"Current position of the indicator within the TrackBar!.")]
```

Comment:
Please check all strings for incorrect search and replace

Reply:
Reverted